### ISSUE TITLE
QHeaderView.setResizeMode in PySide2

### ISSUE THREAD

**Issue by finmacmini** [mottosso/Qt.py#139]
Hey Marcus,

So I've been using your Qt.py shim to allow compatibility between PySide and PySide2 -- which has been working great -- thanks for your work. The only thing I noticed that was missing (for our scripts) was the deprecated, but still present in Qt 5 QHeaderView.setResizeMode. I assume PySide2 decided not to include it since it's deprecated.

I added a quick monkey patch to my code to solve the issue (since setSectionResizeMode is the same as setResizeMode).
```python
if Qt.__binding__ in ('PySide2', 'PyQt5'):
    Qt.QtWidgets.QHeaderView.setResizeMode = Qt.QtWidgets.QHeaderView.setSectionResizeMode
```

Ideally this would be handled by Qt.py itself. I looked through you're code and I wasn't sure the best way to implement it according to your design (and subsequently issue a pull request). So I thought I'd contact you to see if it's easy to add.

Anyway, just thought I'd ask.

Cheers,
Mark

**Comment by mottosso** [comment]
Thanks @finmacmini!

This particular incompatibility has been a popular topic (see links below) and in a nutshell the only reason it isn't in Qt.py itself is because an obscure problem were found in monkey patching the way you illustrate, pull-request [#112](https://github.com/mottosso/Qt.py/pull/112) has more information on this.

Due to the popularity, it's possible we may have to take another look at working around or further prove that it does actually cause problems, but at the moment the current recommended approach is listed in [CAVEATS.md](https://github.com/mottosso/Qt.py/blob/master/CAVEATS.md#qtwidgetsqheaderviewsetresizemode).

- https://github.com/mottosso/Qt.py/pull/112
- https://github.com/mottosso/Qt.py/issues/124
- https://github.com/mottosso/Qt.py/issues/109

You can help by either thinking of an alternative workaround or implementation, or somehow finding a way to prove that the obscure error is isolated to this one occasion.

**Comment by mottosso** [comment]
Shortly after typing this up, an idea occurred to me.

Let me know what you think about this.

```python
from Qt import QtWidgets, QtCompat
app = QtWidgets.QApplication(sys.argv)
view = QtWidgets.QTreeWidget()
header = view.header()
QtCompat.setSectionResizeMode(header, QtWidgets.QHeaderView.Fixed)
```

Where `QtCompat` is an added module containing this and other compatibility functionality, away from other modules so as to not have to monkey patch anything.

**Comment by finmacmini** [comment]
Yeah, that would work, and is obvious what's happen.

**Comment by mottosso** [comment]
Implemented in [0.6.0](https://github.com/mottosso/Qt.py/releases/tag/0.6.0).

Let me know if this works for you, @finmacmini!

**Comment by finmacmini** [comment]
Awesome. That was quick. Yep, works great. Thanks.
